README.md

```diff
@@ -47,7 +47,7 @@ Kodi's developers work hard to make it support a large range of devices and oper
 ## How to Contribute
 Kodi is created by users for users and **we welcome every contribution**. There are no highly paid developers or poorly paid support personnel on the phones ready to take your call. There are only users who have seen a problem and done their best to fix it. This means Kodi will always need the contributions of users like you. How can you get involved?
 
-* **Coding:** Developers can help Kodi by **[fixing a bug](https://trac.kodi.tv/)**, adding new features, making our technology smaller and faster and making development easier for others. Kodi's codebase consists mainly of C++ with small parts written in a variety of coding languages. Our add-ons mainly consist of python and XML. For more information, please have a look at our **[contributing guide](docs/CONTRIBUTING.md)**.
+* **Coding:** Developers can help Kodi by **[fixing a bug](https://github.com/xbmc/xbmc/issues)**, adding new features, making our technology smaller and faster and making development easier for others. Kodi's codebase consists mainly of C++ with small parts written in a variety of coding languages. Our add-ons mainly consist of python and XML. For more information, please have a look at our **[contributing guide](docs/CONTRIBUTING.md)**.
```

Comment:
Consider using a relative link:
```md
(../../issues)
```
so that it works correctly for forks.

Reply:
../.. isn't always correct

Reply:
Oh, when is it not correct? Just curious.

Reply:
e.g. https://github.com/xbmc/xbmc